assets/xml/objects/object_ani.xml

```diff
@@ -1,61 +1,63 @@
-        <DList Name="object_ani_DL_004C30" Offset="0x4C30" />
-        <DList Name="object_ani_DL_004E20" Offset="0x4E20" />
-        <DList Name="object_ani_DL_004F08" Offset="0x4F08" />
-        <DList Name="object_ani_DL_005000" Offset="0x5000" />
-        <DList Name="object_ani_DL_005270" Offset="0x5270" />
-        <DList Name="object_ani_DL_005420" Offset="0x5420" />
-        <DList Name="object_ani_DL_005668" Offset="0x5668" />
-        <DList Name="object_ani_DL_005778" Offset="0x5778" />
-        <DList Name="object_ani_DL_005928" Offset="0x5928" />
-        <DList Name="object_ani_DL_005C50" Offset="0x5C50" />
-        <DList Name="object_ani_DL_005D60" Offset="0x5D60" />
-        <Texture Name="object_ani_TLUT_005E58" OutName="tlut_005E58" Format="rgba16" Width="16" Height="16" Offset="0x5E58" />
-        <Texture Name="object_ani_Tex_006058" OutName="tex_006058" Format="ci8" Width="8" Height="8" Offset="0x6058" />
-        <Texture Name="object_ani_Tex_006098" OutName="tex_006098" Format="ci8" Width="8" Height="8" Offset="0x6098" />
-        <Texture Name="object_ani_Tex_0060D8" OutName="tex_0060D8" Format="i8" Width="8" Height="8" Offset="0x60D8" />
-        <Texture Name="object_ani_Tex_006118" OutName="tex_006118" Format="ci8" Width="16" Height="16" Offset="0x6118" />
-        <Texture Name="object_ani_Tex_006218" OutName="tex_006218" Format="ci8" Width="16" Height="16" Offset="0x6218" />
-        <Texture Name="object_ani_TLUT_006318" OutName="tlut_006318" Format="rgba16" Width="16" Height="16" Offset="0x6318" />
-        <!-- <Blob Name="object_ani_Blob_006518" Size="0x380" Offset="0x6518" /> -->
-        <Texture Name="object_ani_Tex_006898" OutName="tex_006898" Format="ci8" Width="16" Height="16" Offset="0x6898" />
-        <Texture Name="object_ani_Tex_006998" OutName="tex_006998" Format="ci8" Width="8" Height="8" Offset="0x6998" />
-        <Texture Name="object_ani_Tex_0069D8" OutName="tex_0069D8" Format="ci8" Width="8" Height="8" Offset="0x69D8" />
-        <Texture Name="object_ani_Tex_006A18" OutName="tex_006A18" Format="ci8" Width="16" Height="16" Offset="0x6A18" />
-        <!-- <Blob Name="object_ani_Blob_006B18" Size="0x800" Offset="0x6B18" /> -->
-        <DList Name="object_ani_DL_007B58" Offset="0x7B58" />
-        <Animation Name="object_ani_Anim_009220" Offset="0x9220" />
-        <Animation Name="object_ani_Anim_009734" Offset="0x9734" />
-        <Animation Name="object_ani_Anim_009D34" Offset="0x9D34" />
-        <Animation Name="object_ani_Anim_00A460" Offset="0xA460" />
-        <Animation Name="object_ani_Anim_00A978" Offset="0xA978" />
-        <Animation Name="object_ani_Anim_00B2B0" Offset="0xB2B0" />
-        <Animation Name="object_ani_Anim_00B8AC" Offset="0xB8AC" />
+        <Animation Name="gAniTreeHangLosingBalanceAnim" Offset="0x7FC" />
+        <Animation Name="gAniFallOverHoldingFootAnim" Offset="0xC14" />
+        <Animation Name="gAniHoldingFootPainWrithingAnim" Offset="0x11CC" />
```

Comment:
`HoldingFootWrithingInPain`, if we're using this style, or `HoldFootAndWritheInPain` if the other. We'd better discuss that.